test/prototype/moe_training/test_scaled_grouped_mm.py

```diff
@@ -249,3 +249,25 @@ def test_emulate_mxfp8_grouped_gemm(M, K, N, num_experts):
     sqnr = compute_error(ref_out, out)
     min_sqnr = 27.0
     assert sqnr >= min_sqnr, f"sqnr {sqnr} is too low, must be >= {min_sqnr}"
+
+
+@pytest.mark.parametrize("M", (1024, 4096))
```

Comment:
nit: might be good to test one case where MKN are the same, and one where they are all different.  If you want to do that and keep # of tests manageable, it would probably be iterating on MKN in one go instead of iterating on each individually.

Reply:
Updated to parameterize M,N,K together and test "all same" and "all different" cases

Reply:
Ok it's updated now.